app/views/ems_object_storage/edit.html.haml

```diff
@@ -0,0 +1,4 @@
+= react('ProviderForm', :providerId => @ems.id.to_s,
+  :redirect => @lastaction == 'show_list' ? ems_object_storage_path : ems_object_storage_path(@ems),
+  :kind => 'storage',
+  :title => ui_lookup(:model => 'ManageIQ::Providers::StorageManager'))
```

Comment:
This will return all storage managers, not just object storage managers which is why I think @iv1111 is seeing AutoSDE in the list

Reply:
@agrare any idea what is the right kind i need to use?